react/src/pages/addParticipants/DataEntryTable.tsx

```diff
@@ -45,19 +45,59 @@ const useTableStyles = makeStyles(theme => ({
     },
 }));
 
-const defaultOptionals = ["notes", "sex", "input_hpf_path"];
+const fallbackColumns = ["notes", "sex", "input_hpf_path"];
+
+function getEnvColumns(): Array<keyof DataEntryRowOptional> {
+    if (process.env.NODE_ENV === "development") {
+        const envCols = process.env.REACT_APP_DEFAULT_OPTIONAL_COLUMNS;
+        if (envCols !== undefined) {
+            const validFields = getDataEntryHeaders().optional;
+            const envFields = envCols.split(",");
+            const usableFields = envFields.filter(
+                field => !!validFields.find(valid => valid === field)
+            );
```

Comment:
The idea was that just this check is exclusive to dev to catch mistakes. Now production builds will always have `[]` and ignore `REACT_APP_DEFAULT_OPTIONAL_COLUMNS`.

Reply:
Ah my bad, I misunderstood.